src/main/java/com/klarna/hiverunner/builder/HiveShellBase.java

```diff
@@ -100,7 +102,21 @@
     }
     
     private List<Object[]> executeStatementWithCommandShellEmulation(String hql) {
-      return hiveServerContainer.executeStatement(commandShellEmulation.transformStatement(hql));
+      String transformedHql = commandShellEmulation.transformStatement(hql);
+      if (commandShellEmulation.isImportFileStatement(transformedHql)) {
+        return importScript(transformedHql);
+      }
+      return hiveServerContainer.executeStatement(transformedHql);
+    }
+
+    private List<Object[]> importScript(String hql) {
+      String[] tokens = hql.trim().split("\\s+");
+      String fileName = null;
+      if (tokens.length == 2) {
+        fileName = tokens[1];
+      }
+      execute(new File(fileName));
```

Comment:
Do you want to do this if fileName is null? Probably not? Maybe move this into the above if statement. Is it OK to fail silently (i.e. do nothing) if fileName is null? When would this happen?